dts/arm/st/f4/stm32f412.dtsi

```diff
@@ -34,6 +34,20 @@
 			};
 		};
 
+		i2s1: i2s@40013000 {
```

Comment:
This file includes `st/f4/stm32f411.dtsi`, by adding this node you're duplicating i2s1 definition.